snap/snapcraft.yaml

```diff
@@ -258,12 +258,21 @@ parts:
   qemu:
     override-pull: |
       versions_url=https://raw.githubusercontent.com/kata-containers/runtime/${SNAPCRAFT_PROJECT_VERSION}/versions.yaml
-      branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
-      url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.url)"
+      case "$(arch)" in
+        "aarch64")
+          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.architecture.aarch64.tag)"
+          url="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu.url)"
+        ;;
+
+        *)
+          branch="$(curl -sSL ${versions_url} | yq r - assets.hypervisor.qemu-lite.branch)"
```

Comment:
Only `x86_64` uses `qemu-lite`, as implied by the `QEMUCMD` variables in https://github.com/kata-containers/runtime/tree/master/arch/. So I'm rather confused by how `ppc64le` and `s380x` snaps are currently handled @devimc.

Do we need entries for the following in the [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml)?:

- `assets.hypervisor.qemu.architecture.s390x`
- `assets.hypervisor.qemu.architecture.ppc64le`

That would allow only `x86_64` to be special-cased here as the other arches could be handled using something like `assets.hypervisor.qemu.architecture.$(arch).tag`.

Reply:
@jodh-intel `qemu-lite` works on all architectures as long as `pc-lite` isn't being used